source/data-modeling/field-types.txt

```diff
@@ -0,0 +1,558 @@
+  ``BigDecimal`` returns the value unsigned.
+- ``Decimal128`` maintains trailing zeroes, but ``BidDecimal`` does not.
+  Because of this, retrieving ``Decimal128`` values from the database as
+  ``BigDecimal`` might result in a loss of precision.
+
+.. note::
+   
+   When you set the ``Mongoid.map_big_decimal_to_decimal128`` option to ``false``
+   and store a ``BigDecimal`` into an untyped field, you cannot query the field 
+   as a ``BigDecimal``. Because the value is stored as a string, querying 
+   the untyped field for a ``BigDecimal`` value does not find the value in the
+   database. To find the value, you must first convert the query value to a string.
+
+   You can avoid this issue by specifying the field as a ``BigDecimal`` type,
+   rather than untyped.
+
+Specify Fields with Strings or Symbols
+---------------------------------------
+
+You can use strings or symbols to specify certain field types in {+odm+}, instead of
+using their class names. The following example specifies the ``state`` field by
+using the class name, a string, and a symbol:
+
+.. code-block:: ruby
+
+   class Order
+     include Mongoid::Document
+     
+     # Class Name
+     field :state, type: Integer
+
+     # Symbol
+     field :state, type: :integer
+     
+     # String
+     field :state, type: "integer"
+   end
+
+The following field types can be specified as a string or a symbol:
+
+.. list-table::
+   :header-rows: 1
+
+   * - Class Name
+     - Symbol
+     - String
+
+   * - ``Array``
+     - ``:array``
+     - ``"Array"``
+   
+   * - ``BigDecimal``
+     - ``:big_decimal``
+     - ``"BigDecimal"``
+   
+   * - ``BSON::Binary``
+     - ``:binary``
+     - ``"BSON::Binary"``
+   
+   * - ``Mongoid::Boolean``
+     - ``:boolean``
+     - ``"Mongoid::Boolean"``
+   
+   * - ``Date``
+     - ``:date``
+     - ``"Date"``
+   
+   * - ``DateTime``
+     - ``:date_time``
+     - ``"DateTime"``
+   
+   * - ``Float``
+     - ``:float``
+     - ``"Float"``
+   
+   * - ``Hash``
+     - ``:hash``
+     - ``"Hash"``
+   
+   * - ``Integer``
+     - ``:integer``
+     - ``"Integer"``
+   
+   * - ``BSON::ObjectId``
+     - ``:object_id``
+     - ``"BSON::ObjectId"``
+   
+   * - ``Range``
+     - ``:range``
+     - ``"Range"``
+   
+   * - ``Regexp``
+     - ``:regexp``
+     - ``"Regexp"``
+   
+   * - ``Set``
+     - ``:set``
+     - ``"Set"``
+   
+   * - ``String``
+     - ``:string``
+     - ``"String"``
+   
+   * - ``StringifiedSymbol``
+     - ``:stringified_symbol``
+     - ``"StringifiedSymbol"``
+   
+   * - ``Symbol``
+     - ``:symbol``
+     - ``"Symbol"``
+   
+   * - ``Time``
+     - ``:time``
+     - ``"Time"``
+
+Custom Field Types
+------------------
+
+You can create custom field types and define how {+odm+} serializes and
+deserializes them. To create a custom field type, define a class that
+implements the following methods:
+
+- ``mongoize``, which takes an instance of your custom type and converts it to
```

Comment:
```suggestion
- ``mongoize()``, which takes an instance of your custom type and converts it to
```

Reply:
applies to all method references

Reply:
I originally left off the `()` since it seems like you can call methods without it in mongoid, but it makes sense to add them for consistency

Reply:
It doesnt look like you added the parentheses